cms/settings/dev.py

```diff
@@ -28,7 +28,7 @@
 # Enable Wagtail's style guide in Wagtail's settings menu.
 # http://docs.wagtail.io/en/stable/contributing/styleguide.html
 INSTALLED_APPS += ["wagtail.contrib.styleguide"]  # noqa: F405
-
+INSTALLED_APPS += ["django_migration_linter"]
```

Comment:
Nit: Add them on like 30?
```py
INSTALLED_APPS += ["wagtail.contrib.styleguide", "django_migration_linter"]  # noqa: F405
```

Reply:
I remember we did try this before. Tried again now but due to some things to do with how things get initialised, this doesn't work.